section_to_course/tests/test_utils.py

```diff
@@ -0,0 +1,48 @@
+"""
+Tests utility functions for section_to_course.
+"""
+
+from common.djangoapps.student.tests.factories import UserFactory
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, BlockFactory
```

Comment:
You know this, but let's make it visible - these are integration tests, so we should add a TODO to GitHub Actions to set up running them properly in the CI. Running tests manually is not maintainable. This will be especially important when upgrading the platform to Olive/Palm.

If we keep the `upgrade-python-requirements.yml`, we can ensure that the compatibility with the `master` branch of `edx-platform` is tested every week to catch any breaking changes in time.